x/superfluid/keeper/twap_price.go

```diff
@@ -34,6 +37,11 @@ func (k Keeper) SetOsmoEquivalentMultiplier(ctx sdk.Context, epoch int64, denom
 }
 
 func (k Keeper) GetSuperfluidOSMOTokens(ctx sdk.Context, denom string, amount sdk.Int) sdk.Int {
+	// If the denom is a concentrated liquidity token, we just strip the position data from the denom
+	if strings.HasPrefix(denom, cltypes.ClTokenPrefix) {
+		index := strings.LastIndex(denom, "/")
+		denom = denom[:index+1]
+	}
```

Comment:
Added here (plus refactored to return errors, since it oddly kept them internal before) https://github.com/osmosis-labs/osmosis/pull/4884/commits/2d4064927a6ba111493aa2d9a585d7c4a56b951f